### PR TITLE
Homework31

### PR DESCRIPTION
**Pull request by cinck**
Hi!

Please check my homework.

### CODE REVIEW

File: homework31/hw31_task1.py [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+import random
+
+
+def random_url():
+    """Returns random URL from list"""
+    urls = [
+        "https://google.com",
+        "https://facebook.com",
+        "https://twitter.com",
+        "https://amazon.com",  # never responds for some reason
```
Comment (acman) [inline homework31/hw31_task1.py:13]: probably, protection is set from bots

File: homework31/hw31_task1.py [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+        "https://apple.com"
+    ]
+    return random.choice(urls)
+
+
+def get_response_from(url: str):
+    """Returns request.get() object if connection successful"""
```
Comment (acman) [inline homework31/hw31_task1.py:20]: I suppose it returns requests.Response

File: homework31/hw31_task2.py [hunk 3]
```diff
@@ -0,0 +1,102 @@
...
+def get_city_data(city_name: str):
+    """
+    Returns city data from UR as Dict() or None if unsuccessful
+    :param city_name: city name
+    :return:
+    """
+    try:
+        result = requests.get(
+            url="https://geocoding-api.open-meteo.com/v1/search",
+            params={
```
Comment (acman) [inline homework31/hw31_task2.py:15]: 👍

File: homework31/hw31_task2.py [hunk 4]
```diff
@@ -0,0 +1,102 @@
...
+    elif code in range(41, 50):
+        return "foggy"
+    elif code in range(51, 60):
+        return "drizzly"
+    elif code in range(61, 70):
+        return "rainy"
+    elif code in range(71, 80):
+        return "snowy"
+    elif code in range(80, 100):
+        return "trying to kill you!"
```
Comment (acman) [inline homework31/hw31_task2.py:48]: lol

File: homework31/hw31_task1.py [hunk 5]
```diff
@@ -0,0 +1,47 @@
...
+        "https://google.com",
+        "https://facebook.com",
+        "https://twitter.com",
+        "https://amazon.com",  # never responds for some reason
+        "https://apple.com"
+    ]
+    return random.choice(urls)
+
+
+def get_response_from(url: str):
```
Comment (acman) [inline homework31/hw31_task1.py]: I think it is clear to name 
```suggestion
def get_response_from_url(url: str):
```

File: homework31/hw31_task2.py [hunk 6]
```diff
@@ -0,0 +1,102 @@
...
+    else:
+        return "unknown"
+
+
+def get_weather_in(city_name: str):
```
Comment (acman) [inline homework31/hw31_task2.py]: nit:
```suggestion
def get_weather_by_city_name(city_name: str):
```

**Review by acman — APPROVED** [review]
GJ! Nice structure

File: homework31/hw31_task1.py (hunk 2 above)
Comment (cinck) [reply homework31/hw31_task1.py:20]: well, yes, indeed ))

File: homework31/hw31_task1.py (hunk 5 above)
Comment (cinck) [reply homework31/hw31_task1.py]: True. It will work.

File: homework31/hw31_task2.py (hunk 6 above)
Comment (cinck) [reply homework31/hw31_task2.py]: Fair enough. Thanks.

**Pull request merged**